### PR TITLE
major cleanups/fixes and many more i forgot

### PR DESCRIPTION
**Pull request by lenforiee**
Hi i have cleaned up the code lil bit, i also fixed some python funcs overwrite, not used f-strings, added basic anticheat class, and fixed major issue in asahi that could ponetially leak someones credentials, string write now can allow language specific signs such as `ą`, and cleaned the asahi curse (menu callbacks) ddddd, enjoy :p

### CODE REVIEW

**Comment by RealistikDash** [comment]
Yes

**Comment by tsunyoku** [comment]
ok so all seems good but i’m not a fan of this anticheat object, i probably will not use it or will massively adjust it before i do

**Comment by tsunyoku** [comment]
example with my problem with the object is some hard coded md5 checks, not really a fan. also not quite sure why you have some version checks in there for versions before 20210125 as they will literally get rejected anyways. remove the anticheat object in a commit and i’ll merge it :p

File: objects/anticheat.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
+
+class Anticheat:
```
Comment (atteure) [inline objects/anticheat.py:2]: Anticheat object has no use right now

**Review by atteure — CHANGES_REQUESTED** [review]
just remove anticheat object, then lgtm

**Comment by lenforiee** [comment]
done

**Pull request merged**